<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReflowController</class>
 <widget class="QMainWindow" name="ReflowController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ReflowController</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="1,2" columnstretch="2,3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Droid Sans Armenian&quot;;</string>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="leadedButton">
        <property name="styleSheet">
         <string notr="true">font: 16pt &quot;Droid Sans Armenian&quot;;</string>
        </property>
        <property name="text">
         <string>Leaded Solder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="unleadedButton">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>UnLeaded Solder</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pivaHeatButton">
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Piva HEAT</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="styleSheet">
         <string notr="true">font: 20pt &quot;Droid Sans Armenian&quot;;</string>
        </property>
        <property name="text">
         <string>Status:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QProgressBar" name="powerOutputProgressBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="value">
         <number>20</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="textDirection">
         <enum>QProgressBar::BottomToTop</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="powerOutputLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Power Output: -- %</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="AnalogGauge" name="temperatureGauge" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>1</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="temperatureLabel">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Droid Sans Armenian&quot;;</string>
        </property>
        <property name="text">
         <string>-- Â°C</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AnalogGauge</class>
   <extends>QWidget</extends>
   <header>analoggauge.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
